### PR TITLE
Feature/pelagos 4595 allow to submit large file submission

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: assets/static/js/dataset-submission.js [hunk 1]
```diff
@@ -425,6 +425,22 @@ $(function() {
         }
     });
 
+    $("#datasetFilePath").on("keyup change", function() {
+        console.log('made it here');
+        console.log($(this).val());
```
Comment (mickel1138) [inline assets/static/js/dataset-submission.js:430]: Remove

File: templates/DatasetSubmission/index.html.twig [hunk 2]
```diff
@@ -80,8 +80,9 @@
                 Sorry, but you can only create or edit dataset submissions for datasets associated with your research groups.
         </div>
         {% include 'DatasetSubmission/form.html.twig' %}
+
     {% elseif datasetSubmission.status == constant('App\\Entity\\DatasetSubmission::STATUS_COMPLETE')
-              and (datasetSubmission.getFileset.isDone == false)
+              and (datasetSubmission.getFileset.isDone ?? true == false)
```
[the remark this answers is not in the document]
Comment (mickel1138) [reply templates/DatasetSubmission/index.html.twig:85]: This was already here, I just fixed it. If getFileset is null, the function doesn't work.

[the remark this answers is not in the document]
Comment (praneethpr) [reply templates/DatasetSubmission/index.html.twig:85]: Shouldn't be this like `if (datasetSubmission.getFileset and (datasetSubmission.getFileset.isDone == false)`

[the remark this answers is not in the document]
Comment (mickel1138) [reply templates/DatasetSubmission/index.html.twig:85]: That looks nicer, but we have to think about why it was there. If there is not fileset, this `and` should be ignored.

**Review by fightingtexasaggie — APPROVED** [review]
reviewed, tested, all seems to work as expected now

**Pull request merged**